vmdb/app/models/condition.rb

```diff
@@ -29,7 +29,7 @@ def applies_to?(rec, inputs={})
   end
 
   def self.conditions
-    find(:all).collect {|c| c.expression}
+    pluck(:expression)
```

Comment:
I think expression is a serialized column.  Does pluck do the serialization or does it do raw SQL value.  Trying to keep that in mind when I review pluck changes.  I'm not sure they call the getters if they are overridden.